perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -966,7 +966,7 @@ public ServiceAttributes call(ApiCaller ac, Deserializer parms) throws PerunExce
 
 		@Override
 		public ServicesPackage call(ApiCaller ac, Deserializer parms) throws PerunException {
-			return ac.getServicesPackageById(parms.readInt("servicesPackageId"));
+			return ac.getServicesManager().getServicesPackageById(ac.getSession(), parms.readInt("servicesPackageId"));
```

Comment:
Same goes for all methods working with service packages, since facility admin is the one to get/assign service packages to the facility and perun admin manages those packages.